webapp/src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { Icon, IconType } from './components/Icon/Icon';
```

Comment:
As discussed by chat, we should start organizing files and folders since the beggining so we don't face refactor issues in the future. We could have a folder named `pages` inside it, we could have the views related of each page. And inside each page folder, the needed `tsx`, `css` files and its own components/assets etc.

```txt
/src
- /pages
- App.tsx
- /landing
-- Landing.tsx
-- Landing.css
-- /components
--- (...)
...
```

Also, although for now is good, we shouldn't need any `App.css` and may have a `index.css `next to `index.tsx` so we have common styles there (for `:root`, `body` etc.)

Reply:
Done :+1:

Reply:
We can talk about the folder structure during next meeting, because I have some doubts. 

- A folfer for each single page?
- Inside this folder, at least 2 files: page.tsx and pase.css?
- So, a .css for each page?
- Variables in css like, for example, `--main-bg-color: black`, would they be in each file or just in the index.css?
- Where should we define our basic pallette and fonts?

Reply:
@fizcamposancos, sure, we could talk about it.

Currently planning to do something like this:

```
/src
- /pages
--  App.tsx
--  /Landing
---   Landing.tsx
---   Landing.css
---   index.tsx
---   /components
----   ...
--  /SomePage
---   SomePage.tsx
---   SomePage.css
---   index.tsx
---   ...
- /shared
--  /components
---   /Icon
----   Icon.tsx
----   index.tsx
----   /assets
-----    ...
---  /Button
----   Button.tsx
----   Button.css
----   index.tsx
- /assets
--  ...
...
```

The `pages` directory is for the views, the `shared` is for the shared components and files, and the `assets` is for other files.

What do we think about this structure?

Reply:
I really like that structure @gbudau! we can integrate a first approach like that and discuss in the next meeting about details. What @fizcamposancos says about `css` is also very important :+1:

Reply:
AFAIK we can use `@import "xxx.css";`in css, in case we want to reduce code duplication, and still have one .css file per component. Although I'm not sure how much do we need a particular css per component.